docs/additional-package-support.md

```diff
@@ -5,15 +5,40 @@
 ------
 
 # PWA 
+## Screenshot
+
+![PC](https://i.loli.net/2019/02/09/5c5e15567c52d.jpg)
+![Mobile](https://i.loli.net/2019/02/09/5c5e1556af49b.png)
 
+------
 !> Since v1.2
 
 To have the PWA feature for `theme-melody`, you need to do the following things:
 
 1. Go to the hexo site folder
 2. `npm install hexo-offline --save` or `yarn add hexo-offline`
-3. Modify the `_config.yml` -> see the doc of [hexo-offline](https://github.com/JLHwung/hexo-offline)
-4. Set the `melody.yml`:
+3. Modify `_config.yml`
+Add the following to the site `_config.yml`.
+
+```yaml
+# offline config passed to sw-precache.
+offline:
+  maximumFileSizeToCacheInBytes: 10485760 # The maximum file size cached, in bytes
+  staticFileGlobs:
+    - public/**/*.{js,html,css,png,jpg,gif,svg,webp,eot,ttf,woff,woff2}
+  # Static file collection, if your site uses files such as webp format, please add the file type.
+  stripPrefix: public
+  verbose: true
+  runtimeCaching:
+    # CDNs - should be cacheFirst, since they should be used specific versions so should not change
+    - urlPattern: /* # If you need to load CDN resources, please configure this option. If not, you can configure it.
+      handler: cacheFirst
+      options:
+        origin: your_websie_url # can be replaced with your url
+```
+For more details, please see the official documentation for [hexo-offline] (https://github.com/JLHwung/hexo-offline)
```

Comment:
![image](https://user-images.githubusercontent.com/12621342/52514699-2073a880-2c4f-11e9-8e73-72b6d203f87c.png)
There is an extra space here